src/pages/docs/run-anywhere/smallstep-platform-installation-gcp.mdx

```diff
@@ -0,0 +1,174 @@
+# Smallstep Platform Installation - GCP
+
+## **Setting up GCP Infrastructure**
+
+This document describes how to install the Smallstep Platform on GCP infrastructure.
+
+### Dependencies
+
+- [gcloud](https://cloud.google.com/sdk/docs/downloads-interactive)
+- [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [KOTS](https://kots.io/kots-cli/getting-started/#how-to-install)
+- [linkerd](https://linkerd.io/2.10/getting-started/#step-1-install-the-cli)
+- [step](https://smallstep.com/docs/step-cli/installation)
+
+### Terraform
+
+Create a GCP project and create infrastructure using instructions in the shared terraform repo.
+
+If you already have a Terraform environment set up for your account, add the [run-anywhere-terraform](https://github.com/smallstep/run-anywhere-terraform/tree/main/gcp) module to your configuration and follow the steps listed in its README, skipping the steps involving state creation. If you are new to Terraform, follow the [run-anywhere-terraform](https://github.com/smallstep/run-anywhere-terraform/tree/main/gcp) module’s README in its entirety: it will instruct you how to set up a Terraform [state](https://www.terraform.io/language/state) in your GCP account, instantiate the [module](https://www.terraform.io/language/modules/syntax), and run the subsequent Terraform [apply](https://www.terraform.io/cli/commands/apply).
```

Comment:
If possible, I think we should move the caveats in this section into the Terraform repo README and just ask that they follow the instructions there.

Reply:
On second thought, how about we inline all the terraform docs from that repo into this doc, so everything is in one place?

Reply:
I'll pull those over once I have them finalized (probably later today / tomorrow morning)